package.json

```diff
@@ -12,8 +12,9 @@
     "./src"
   ],
   "dependencies": {
+    "@aws-sdk/client-dynamodb": "3.113.0",
```

Comment:
I'm pinning to the same versions that `@shelf/jest-dynamodb` uses because I was encountering errors with that library before doing so.